tests/testthat/test-ols-var.R

```diff
@@ -1,12 +1,25 @@
+set.seed(1)
 test_that("lm and sandwhich variance - 4 dec.pl.", {
-  n <- 2e4
-  X <- rnorm(n, 0, 1)
-  y <- 2 + X * 1 + rnorm(n, 0, 10)
-  lm_fit <- lm(y ~ X)
-  sandwich_var <- comp_sandwich_var(X, lm_fit$residuals)
+  n <- 1e5
+  X <- stats::rnorm(n, 0, 1)
+  y <- 2 + X * 1 + stats::rnorm(n, 0, 10)
+  lm_fit <- stats::lm(y ~ X)
+  sandwich_qr_var <- comp_sandwich_qr_var(lm_fit)
   expect_equal(
-    round(broom::tidy(lm_fit)$std.error, 3),
-    round(sqrt(diag(sandwich_var)), 3)
+    round(broom::tidy(lm_fit)$std.error, 4),
+    round(sqrt(diag(sandwich_qr_var)), 4)
```

Comment:
We can change the rounding to 7 dec. pl. Our test is more robust now since our function is numerically better